vms/shift/tests/test_unit.py

```diff
@@ -410,3 +412,166 @@ def test_model_representation(self):
 
         # Check correctness
         self.assertEqual(str(vol_shift_in_db), 'job-name - 2050-05-24 - Son')
+
+
+class EditRequestModelTests(TestCase):
```

Comment:
Docstrings are missing in tests of this model.

Reply:
added